Dockerfile

```
@@ -2,12 +1,20 @@
+FROM registry.access.redhat.com/ubi9/ubi-minimal:9.5-1742914212
 SHELL ["/bin/bash", "-c"]
 USER root
+# Install snappy for kafka-python
+RUN cd /tmp && \
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)